tests/unit/dataactbroker/test_generation_helper.py

```diff
@@ -1,13 +1,283 @@
 import pytest
```

Comment:
----
NOTE: I didn't get to reviewing completeness/coverage of tests, and ran out of time on the review here. But there's enough above to go on.

----